install.sh

```diff
@@ -139,10 +141,12 @@ popd
 
 # Build packet forwarder
 if [ ! -d packet_forwarder ]; then
-    git clone https://github.com/TheThingsNetwork/packet_forwarder.git
+    git clone -b legacy https://github.com/TheThingsNetwork/packet_forwarder.git
     pushd packet_forwarder
 else
     pushd packet_forwarder
+    git fetch origin
+    git checkout legacy
     git pull
```

Comment:
`git pull` is no longer needed since we do `git fetch` combined with `git reset --hard`